### PR TITLE
add pytest.ini and fix docs

### PR DESCRIPTION
**Pull request by jin-s13**

### CODE REVIEW

File: mmpose/models/backbones/litehrnet.py [hunk 1]
```diff
@@ -792,15 +789,13 @@ def __init__(self,
                  conv_cfg=None,
                  norm_cfg=dict(type='BN'),
                  norm_eval=False,
-                 with_cp=False,
-                 zero_init_residual=False):
+                 with_cp=False):
```
Comment (ly015) [inline mmpose/models/backbones/litehrnet.py:792]: zero_init_residual was unused?

**Review by ly015 — APPROVED** [review]

File: mmpose/models/backbones/litehrnet.py (hunk 1 above)
Comment (jin-s13) [reply mmpose/models/backbones/litehrnet.py:792]: Not used param.

**Comment by ly015** [comment]
Please test running pytest locally with this pytest.ini

File: pytest.ini [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[pytest]
```
Comment (innerlee) [inline pytest.ini:1]: no need to add a new file. see https://docs.pytest.org/en/6.2.x/customize.html#setup-cfg

Comment (jin-s13) [reply pytest.ini:1]: However, here is a warning. 

```
Warning
Usage of setup.cfg is not recommended unless for very simple use cases. .cfg files use a different parser than pytest.ini and tox.ini which might cause hard to track down problems. When possible, it is recommended to use the latter files, or pyproject.toml, to hold your pytest configuration.
```

Comment (innerlee) [reply pytest.ini:1]: I thought ours belong to the simple case haha. Anyway keep or not, either is okay to me.

**Pull request merged**